ec3730/Models/DataUsageModel.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  DataUsageModel.swift
+//  ec3730
+//
+//  Created by Ahmad Azam on 21/08/2022.
+//  Copyright © 2022 Zachary Gorak. All rights reserved.
+//
+
+import Foundation
+
+extension SystemDataUsage {
+    static var wifiCompelete: String {
+        let usage = dataUsage.wifiSent + dataUsage.wifiReceived
+        return humanReadableByteCount(bytes: usage)
+    }
+
+    static var wwanCompelete: String {
+        let usage = dataUsage.wirelessWanDataSent + dataUsage.wirelessWanDataReceived
+        return humanReadableByteCount(bytes: usage)
+    }
+
+    static var dataUsage: DataUsageInfo {
+        SystemDataUsage.getDataUsage()
```

Comment:
Is this blocking or slow at all?

Are these numbers reset or anything between app launches?

Reply:
It is not slow these number updated on realtime so computed property is better but I think we can keep it as a static variable and reload it on reload function to have new update values

Reply:
Got it. Just wanted to make sure we shouldn't be passing this work to a Task or something